grants_tagger/evaluate_model.py

```diff
@@ -81,5 +81,7 @@ def evaluate_model(model_path, data_path, label_binarizer_path, threshold):
 
     # comma indicates multiple threshold to evaluate against
     if "," in threshold:
-        threshold = threshold.split(",")
+        threshold = [float(t) for t in threshold.split(",")]
+    else:
+        threshold = float(threshold)
```

Comment:
not sure how likely this scenario is, so feel free to ignore, but if someone inputs `threshold = '0.4, 0.5, '` (i.e. a comma at the end) this will break. Just thinking this might come up if you are reading from a file maybe?

Reply:
interesting but i am tempted to say that this case is not supported for now